### PR TITLE
Support for multiple slack clients

### PR DESCRIPTION
**Pull request by juzerali**
Fixes razorpay/api#7354

Impacted components: razorpay/api#7367

### CODE REVIEW

**Comment by juzerali** [comment]
@StSarc please review.

File: src/config/config.php [hunk 1]
```diff
@@ -1,126 +1,130 @@
...
+
+        // Allow Markdown in all fields
+        // 'markdown_in_attachments' => ['pretext', 'text', 'title', 'fields', 'fallback']
+
+        // Use non-default connection for slack,
+        // if set to null, it will use the default connection
+        'queue' => null,
+    ],
+
+    'clients'   =>  [],
```
Comment (captn3m0) [inline src/config/config.php]: write some comments about how this works

File: src/config/config.php [hunk 2]
```diff
@@ -1,126 +1,130 @@
...
-    // 'markdown_in_attachments' => ['text', 'title']
-
-    // Allow Markdown in all fields
-    // 'markdown_in_attachments' => ['pretext', 'text', 'title', 'fields', 'fallback']
     'is_slack_enabled' => true,
 
-    // Use non-default connection for slack,
-    // if set to null, it will use the default connection
-    'queue' => null,
+    'defaults'  =>  [
```
Comment (captn3m0) [inline src/config/config.php]: This looks like a breaking change, are we doing a migration guide along with a major version bump?

Comment (juzerali) [reply src/config/config.php]: Didn't think about that. Where should I put the migration guide? will it be in `README.md` file?

**Pull request merged**